test/e2e/testsuite/testsuite.go

```diff
@@ -324,12 +325,18 @@ func InitializeScenario(s *godog.ScenarioContext) {
 		ExecuteCommandWithExpectedExitStatus)
 	s.Step(`^execut(?:e|ing) single crc (.*) command (.*)$`,
 		ExecuteSingleCommandWithExpectedExitStatus)
+	s.Step(`^executing podman command (.*) (succeeds|fails)$`,
+		ExecutingPodmanCommandSucceedsFails)
 	s.Step(`^ensuring CRC cluster is running (succeeds|fails)$`,
 		EnsureCRCIsRunningSucceedsOrFails)
```

Comment:
Exec as single command on `setup`

Reply:
Podman commands are all "single". Maybe it'd have been good to call the function `ExecutingSinglePodmanCommandSucceedsFails`, but it seemed better to omit it.